relay-general/src/store/normalize/contexts.rs

```diff
@@ -69,18 +72,35 @@ fn normalize_runtime_context(runtime: &mut RuntimeContext) {
     }
 }
 
+fn get_windows_version(description: &str) -> Option<&str> {
+    if let Some(captures) = OS_WINDOWS_REGEX1.captures(description) {
+        captures.name("version").map(|m| m.as_str())
+    } else if let Some(captures) = OS_WINDOWS_REGEX2.captures(description) {
+        captures.name("version").map(|m| m.as_str())
+    } else {
+        None
+    }
+}
+
+#[allow(dead_code)]
+// TODO use this to add a tag `android.api` to the message (not yet 100% decided)
```

Comment:
Just so that my concerns are written down somewhere: We do not yet know whether it makes sense to store this as a tag or whether we would want S&S to create a new column for this. From my understanding, s&s is already pretty unhappy with tags and their broad applicability in general. In this case I see potential to store this more efficiently than as a string in clickhouse. However at the same time this tag will be there for not a lot of events at all.

Reply:
What's the right way to drive this? Is this something @fpacifici can give us guidance?
It’s an integer (currently between 1 and 30), around once a year there’s a new one. And it’s valuable information to Android devs

Reply:
I just realized that the android API version is not persisted in the event at all, so we need another PR where this function is used to populate some event context

Reply:
@bruno-garcia to get this kickstarted, where do we store this in regular android events right now? is that some context, a tag, ...? 

Imho, we should strive to put this in the same place as the Android SDK does, and depending on where that is and where it can be derived from, we can make this either a derived tag, or part of event normalization.

Reply:
Yes, I agree. At this time we're not setting this anywhere from the Android SDK.
It only sends the Android version such as:
![image](https://user-images.githubusercontent.com/1633368/131881727-35e851d0-aaab-42aa-acac-babddb61a49d.png)

I started a thread with the team to discuss this and I'll follow up through a new request if that makes sense. We can close this thread (I often don't read notifications from merged PRs).